adr/adr-001.md

```diff
@@ -0,0 +1,28 @@
+# ADR-001: Exempt delegations as a transitionary mechanism for safe adoption of liquid staking.
+
+## Context 
+
+Liquid proof of stake may exacerbates the principal agent problem that exists at the heart of the delegated proof of stake system. The core of the problem is that validators do not actually own the stake that is delegated to them. This leaves the open to perverse incentives to attack the consensus system.
+
+Cosmos introduced the idea of min self bond in the staking. This creates a minimum amount of stake the must be bonded by the validators operator key. 
+
+This feature has very little effect on the behavior of delegates.
+
+## Proposal:
+
+The liquid staking system will introduce a new system. Delegator can issue a transaction that exempts their delegation to a specific validator from liquid staking. Once this transaction is issued, the SharesToToken tx will fail for the delegation. 
+
+Furthermore there will be a governance controlled parameter  that will restrict the total number of tokenized shares issued for each validator by the sum of the exempt delegations multiplied by a scalar value.
+
+For example, if governance has set value to 10.  Once a delegation exempts 1000 stake from the liquid staking system, then then other delegators may chose to issue up 10,000 tokenized shares with the tokens to shares transactions.
+
+If the maximum amount of tokenized shares are issued, the exempt delegation cannot be undelegated unless the validator chooses to leave the active validator set.
```

Comment:
Does it mean that if the maximum amount is not issued, the exempt delegation can be undelegated? If so, the delegators will issued an epsilon less than the maximum and the exempt delegation will not play its role.